tests/unity/example_test/src/example_test.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#ifndef __EXAMPLE_TEST_H
```

Comment:
Remove `__` prefix in all the header guards, AFAICS the style used is like `AT_CMD_PARSER_H_`

Reply:
or the longer `ZEPHYR_INCLUDE_`